### DISCUSSION TITLE
Attachments not displayed on FO

### DISCUSSION THREAD

**Discussion by virgileJT** [incidentbot/incidentbot#78] [General]
I can't have the attachment (image) properly displayed in the FO, but get a 401 "Missing Authorization Header" error.

So far, I can see the BO requires a token
backend/bot/api/routes/iincident.py l147 :
@jwt_required()
def get_delete_item_by_id(incident_id, id):

but there is no token sent when displaying the picture on the FO
frontend/src/incident/Signle-incidents.js l678 :
{imgData.map((item) => (
<img src={${item.img}} alt={item.title} loading="lazy" />

I don't know what is the good practice here, maybe rewrite the REACT part to get attachments one by one ?

**Comment by echoboomer** [comment]
@virgileJT thanks for reporting, transferred this to an issue and I'll check it out.
